.github/workflows/test.yml

```
@@ -0,0 +1,217 @@
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-all
+      - uses: ./.github/actions/setup-test
+
+      - name: Prepare folders
+        shell: bash
+        run: |
+          mkdir -pv test-results
+
+  test-pytest:
+    runs-on: ${{ matrix.os }}
```

actionlint (1.7.4)

property "os" is not defined in object type {}

(expression)